Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -1,7 +1,144 @@
 ﻿using System;
-
+using System.Text.RegularExpressions;
 namespace Tavisca.Bootcamp.LanguageBasics.Exercise1
 {
+    class Equation{
+        private string operand1;
+        private string expectedOperand1;
+        private string operand2;
+        private string expectedOperand2;
+        private string operation;
+        private string result;
+        private string expectedResult;
+        private void setOperand1(string value){
+            operand1=value;
+        }
+        public string getOperand1(){
+            return operand1;
+        }
+        public string getExpectedOperand1(){
+            return expectedOperand1;
+        }
+        private void setOperand2(string value){
+            operand2=value;
+        }
+        public string getOperand2(){
+            return operand2;
+        }
+        public string getExpectedOperand2(){
+            return expectedOperand2;
+        }
+        private void setOperation(string value){
+            operation=value;
+        }
+        public string getOperation(){
+            return operation;
+        }
+        private void setResult(string value){
+            result=value;
+        }
+        public string getResult(){
+            return result;
+        }
+        public string getExpectedResult(){
+            return expectedResult;
+        }
+        private void setExpectedValues(){
+            if(operation=="*"){
+                //expectedOperand1
+                if(!operand1.Contains('?'))

```

Comment:
As we have discussed, you should write an method which will give you part that needs to be corrected. Return Type of that method will be an enum which will say whether Operand1/Operand2/Result needs to be corrected.
Once we have what needs to be corrected, we will simply use switch case for further operations. In this way you don't have to write such big if-else ladder